src/plugins/legacy/polygonRoi/polygonLabel.cpp

```diff
@@ -566,15 +566,18 @@ medAbstractData *polygonLabel::createMask(int label, QString &desc)
         {
             for(int j=(int)bounds[by]; j<=(int)bounds[by+1]; j++)
             {
-                QRgb val = imagePolygon.pixel(i,j);
-
-                if (val==qRgb(255,255,255))
+                if (i>0 && i<dimX && j>0 && j<dimY)
```

Comment:
This should be `i >= 0` and `j >= 0` (instead of `>`)

Reply:
Indeed, PR updated